src/pessimisticPatternMatcherUtils.cc

```diff
@@ -0,0 +1,105 @@
+/// -*- LSST-C++ -*-
+
+/*
+ * This file is part of meas_astrom.
+ *
+ * Developed for the LSST Data Management System.
+ * This product includes software developed by the LSST Project
+ * (https:///www.lsst.org).
+ * See the COPYRIGHT file at the top-level directory of this distribution
+ * for details of code ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https:///www.gnu.org/licenses/>.
+ */
+
+#include <cmath>
+#include <iostream>
+#include "ndarray/eigen.h"
+#include "lsst/meas/astrom/pessimisticPatternMatcherUtils.h"
+
+namespace lsst {
+namespace meas {
+namespace astrom {
+
+int check_spoke(double cos_theta_src, double sin_theta_src, ndarray::Array<double, 1, 1> const& ref_ctr,
+                uint16_t ref_ctr_id, ndarray::Array<double, 1, 1> const& proj_ref_ctr_delta,
+                double proj_ref_ctr_dist_sq, ndarray::Array<long int, 1, 1> const& ref_dist_idx_array,
+                ndarray::Array<uint16_t, 1, 1> const& ref_id_array,
+                ndarray::Array<double, 2, 1> const& reference_array, double src_sin_tol) {
+    /// Loop over our candidate reference objects. ref_dist_idx_array is a view into ref_id_array
+    /// and is not the same length as ref_id_array.
```

Comment:
All of the comments should be double slash, not triple.

Reply:
Swear I saw tripled asked for somewhere in the docs but can't find it now. Will change it back.